examples/resources/awscc_ec2_subnet/ec2_subnet_advanced.tf

```diff
@@ -0,0 +1,21 @@
+resource "awscc_ec2_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "awscc_ec2_subnet" "main" {
+  vpc_id     = resource.awscc_ec2_vpc.main.id
+  cidr_block = "10.0.1.0/24"
+  tags = [{
+    key   = "Name"
+    value = "main"
+  }]
+}
+
+resource "awscc_ec2_network_acl" "main" {
```

Comment:
The NACL entry (`AWS::EC2::NetworkAclEntry`) is not available yet unfortunately 
therefore, this example is incomplete at the moment because we are attaching blank NACL without any rules.

I would incline to not add this advanced use-case until we have all relevant resources available.

Reply:
Let's create new GitHub issue so we could track the availability of `AWS::EC2::NetworkAclEntry`

Reply:
```
aws cloudformation describe-type --type RESOURCE --type-name AWS::EC2::NetworkAclEntry | jq ".ProvisioningType"
"NON_PROVISIONABLE"
```

Reply:
> Let's create new GitHub issue so we could track the availability of `AWS::EC2::NetworkAclEntry`

https://github.com/hashicorp/terraform-provider-awscc/issues/955